CONTRIBUTING.md

```diff
@@ -115,14 +115,14 @@ In this case, the parser knows that the previous commit is the parent and that t
 In the case of branches, the first commit on a branch will have a parent that is not the previous commit. 
 Instead, you'll need to include the colon and specify the parent commit. 
 Likewise, if a branch head or a tag points to a commit, then we must include the parenthesis, otherwise, we can leave it out. 
-The most complete example of a spec file is `gitgud/levels/merging/_octopus/test.spec`. 
+The most complete example of a spec file is `gitgud/extras/merging/_octopus/test.spec`. 

```

Comment:
Should be `gitgud/levels/extras/_octopus/test.spec`

Reply:
No `merging` in the path? Alrighty.